src/xdocs/beginning_development.xml

```diff
@@ -27,7 +27,7 @@
         You can find information about development environment preparation here:
         <a href="https://github.com/sevntu-checkstyle/sevntu.checkstyle/wiki/Prepare-Development-Environment-in-Ubuntu-12.04">
         Prepare development environment in Ubuntu</a>.<br/>
-        2. Fork your copy from Checkstyle main project. As it is described
+        2. Fork your copy from Checkstyle upstream project. As it is described
```

Comment:
please make it `Fork Checkstyle upstream project. As it is described`

Reply:
I'm sorry I missed that. Done. @romani